### ISSUE TITLE
Stop uptime reports for nodes with read only cache disk

### ISSUE THREAD

**Issue by scottyeager** [threefoldtech/zos#2134]
Recently I've seen a couple more cases like #1983, where the node logs indicate read only filesystem and no workloads are accepted. I'll include details below for reference.

The thing is that these nodes are still generating uptime reports, and thus minting. Since we only want to mint for nodes that provide usable capacity, I wonder if we can detect this condition in Zos and stop uptime reporting when it happens.

### Node 949
```
[+] provisiond: 2023-11-17T21:28:40Z fatal exiting error="error running integrity checks: unlinkat /var/cache/modules/provisiond/metrics-diff.bolt: read-only file system"
[+] provisiond: 2023-11-17T21:28:40Z error integrity check failed, resetting rrd db error="exit status 1" stderr="failed to create metrics database: failed to open bolt db: open /var/cache/modules/provisiond/metrics-diff.bolt: read-only file system\n"
```

### Node 1941
```
provisiond: 2023-11-17T22:11:41Z error capacity reported stopped unexpectedly error="failed to create rrd slot: failed to apply retnetion policy: write /var/cache/modules/provisiond/metrics-diff.bolt: read-only file system"
```

**Comment by ramezsaeed** [comment]
we should show a relevant message in zui, and stop sending uptime reports until farmers fix that.

**Comment by scottyeager** [comment]
Yes, there's a spot for cache disk status on zui. It can say "Read only" in red.

**Comment by AbdelrahmanElawady** [comment]
for nodes that managed to get the update on mainnet, uptime reports stopped:
![image](https://github.com/threefoldtech/zos/assets/60783742/8c5d6a7f-ecb6-4319-a758-b0402f15d316)
however, the majority didn't get the latest version due to disk issues. which should be addressed here: #2228 #2227
